### PR TITLE
CLOUDP-115605: [MongoCLI] Improve config command to accept opsmanager

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-115605

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

**Issue**: users can only run the config command with `--service ops-manager` or `--service cloud-manager`. If they try to run `config --service opsmanager` they are not able to config mongocli with ops manager

**The goal of this PR is to allow users to use `opsmanager`, `om`, `cloudmanager`, and `cm` with the `mongocli config` command.**

```
./bin/mongocli config --help
Configure settings in a user profile.
All settings are optional. You can specify settings individually by running:
$ mongocli config set --help

You can also use environment variables (MCLI_*) when running the tool.
To find out more, see the documentation: https://docs.mongodb.com/mongocli/stable/configure/environment-variables/.

Usage:
  mongocli config [flags]
  mongocli config [command]

Examples:

  To configure the tool to work with Atlas
  $ mongocli config

  To configure the tool to work with Atlas for Government
  $ mongocli config --service cloudgov
  $ mongocli config --service gov

  To configure the tool to work with Cloud Manager
  $ mongocli config --service cloud-manager
  $ mongocli config --service cloudmanager
  $ mongocli config --service cm

  To configure the tool to work with Ops Manager
  $ mongocli config --service ops-manager
  $ mongocli config --service opsmanager
  $ mongocli config --service om


Available Commands:
  set         Configure specific properties of a profile.
  list        List available profiles.
  describe    Return a specific profile.
  rename      Rename a profile.
  delete      Delete a profile.

Flags:
  -h, --help             help for config
      --service string   Type of MongoDB service. Valid values are cloud, cloudgov, cloud-manager, or ops-manager. (default "cloud")

Global Flags:
  -P, --profile string   Profile to use from your configuration file.

Use "mongocli config [command] --help" for more information about a command.
```

```
./bin/mongocli config --service om
You are configuring a profile for mongocli.

All values are optional and you can use environment variables (MCLI_*) instead.

Enter [?] on any option to get help.

? URL to Access Ops Manager: [? for help]
```

```
./bin/mongocli config --service opsmanager
You are configuring a profile for mongocli.

All values are optional and you can use environment variables (MCLI_*) instead.

Enter [?] on any option to get help.

? URL to Access Ops Manager: [? for help]
```

### CODE REVIEW

File: internal/validate/validate.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func Credentials() error {
 
 	configCMD := "config"
 	if config.BinName() == "atlas" {
-		configCMD += "init"
+		configCMD += " init"
```
Comment (andreaangiolillo) [inline internal/validate/validate.go:109]: drive-by fix. Before this fix we got 
```
atlas projects ls
Error: missing credentials

To set credentials, run: atlas configinit
```

With this fix we have:
```
atlas projects ls
Error: missing credentials

To set credentials, run: atlas config init
```

**Review by fmenezes — APPROVED** [review]
LGTM

**Review by fmenezes — DISMISSED** [review]

**Review by blva — DISMISSED** [review]

**Review by colm-quinn — DISMISSED** [review]
LGTM, thanks for the helpful alias'

File: internal/cli/config/config.go [hunk 2]
```diff
@@ -85,6 +85,33 @@ Enter [?] on any option to get help.
...
+		return nil
+	}
+
+	if opts.Service == "opsmanager" || opts.Service == "om" {
+		opts.Service = config.OpsManagerService
+		return nil
+	}
+
+	if opts.Service != config.OpsManagerService && opts.Service != config.CloudManagerService && opts.Service != config.CloudGovService {
+		return fmt.Errorf("the service '%s' is not supported. Please run 'mongocli config --help' to see the list of available services", opts.Service)
```
Comment (melissamahoney-mongodb) [inline internal/cli/config/config.go:109]:
```suggestion
		return fmt.Errorf("The '%s' service is not supported. Please run 'mongocli config --help' to see the list of available services.", opts.Service)
```

**Review by melissamahoney-mongodb — APPROVED** [review]
Tiny suggestion then LGTM. No need for re-review. :)

File: internal/cli/config/config.go (hunk 2 above)
Comment (andreaangiolillo) [reply internal/cli/config/config.go:109]: Good practice in go is that the error string should not be capitalized or end with a punctuation mark. I am going to address the comment about `service`

**Review by gssbzn — APPROVED** [review]
